lib/elixir/lib/agent.ex

```diff
@@ -74,8 +73,8 @@ defmodule Agent do
   at least initially, or small enough to be sent to the client cheaply. Another
   factor is whether the data needs to be processed atomically: getting the
   state and calling `do_something_expensive(state)` outside of the agent means
-  that the agent's state can be updated in the meantime. This is specially
-  important in case of updates as computing the new state in the client rather
+  that the agent's state can be updated in the meantime. This is especially
+  important when updating the state as computing the new state in the client rather
```

Comment:
I am not sure if the double `-ing` reads well here: "... updating ... computing ...".

Reply:
Hmm I find it easier to make sense of in spite of the double `-ing` personally. But I could see how others would not!